src/quo/components/wallet/summary_info/view.cljs

```diff
@@ -4,6 +4,7 @@
     [quo.components.avatars.user-avatar.view :as user-avatar]
     [quo.components.avatars.wallet-user-avatar.view :as wallet-user-avatar]
     [quo.components.markdown.text :as text]
+    [quo.components.wallet.summary-info.schema :refer [?schema]]
```

Comment:
I don't mind it, but I remember the guidelines having something about prefering `:as` to `:refer`